src/modules/message/message.contoller.ts

```diff
@@ -0,0 +1,23 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { MessageService } from './message.service';
+import { SendMessageTextDto } from './dto/send-message-text.dto';
+import { SendMessageFileDto } from './dto/send-message-file.dto';
+
+@Controller('messages')
+export class MessageController {
+  constructor(private readonly messageService: MessageService) {}
+
+  @Post('/text')
+  async sendMessageText(@Body() sendMessageTextDto: SendMessageTextDto) {
+    console.log('rrrrrr');
+    this.messageService.sendMessageText(sendMessageTextDto);
+  }
+  @Post('/file')
+  async sendMessageFile(
+    @Body()
+    sendMessageFileDto: SendMessageFileDto,
+  ) {
+    console.log('rrrrrr');
+    this.messageService.sendMessageFile(sendMessageFileDto);
+  }
+}
```

Comment:
Para esse endpoint vamos ter que fazer diferente. Vamos receber como multpart/form-data e converter para base64

Reply:
Receber uma string base64 não é ideal, a request fica grande na medida que o tamanho do arquivo for grande

Reply:
Outro motivo também é que para multiplos requests enviando base64 direto pode bloquear o event loop do nodejs

Reply:
Da um olho aqui:
https://docs.nestjs.com/techniques/file-upload